tests/plugins/clipboard.test.ts

```diff
@@ -1292,6 +1292,41 @@ describe("clipboard", () => {
     expect(model.getters.getConditionalStyle(...toCartesian("A2"))).toBeUndefined();
   });
 
+  test("copy paste CF in another sheet => change CF => copy paste again doesn't overwrite the previously pasted CF", () => {
+    const model = new Model();
+    createSheet(model, {});
+    const sheet1Id = model.getters.getSheets()[0].id;
+    const sheet2Id = model.getters.getSheets()[1].id;
+
+    const cf = createEqualCF("2", { fillColor: "#00FF00" }, "cfId");
+    model.dispatch("ADD_CONDITIONAL_FORMAT", {
+      cf,
+      target: target("A1"),
+      sheetId: sheet1Id,
+    });
+
+    model.dispatch("COPY", { target: [toZone("A1")] });
```

Comment:
```suggestion
    model.dispatch("COPY", { target: target("A1") });
```
To be consistent with 3 lines above ^^